libraries/abstractions/ble_hal/include/bt_hal_gatt_client.h

```diff
@@ -104,14 +104,14 @@ typedef enum
     eBTGattcDeviceTimeout,           /* 9  Device timeout                    */
     eBTGattcInvalidControllerOutput, /* 10  An incorrect value was received from HCI */
     eBTGattcSecurityError,           /* 11 Authorization or security failure or not authorized  */
-    eBTGattcDelayedEncryptionCheck,  /*12 Delayed encryption check */
-    eBTGattcErrProcessing            /* 12 Generic error                     */
+    eBTGattcDelayedEncryptionCheck,  /* 12 Delayed encryption check */
+    eBTGattcErrProcessing            /* 13 Generic error                     */
 } BTGattcError_t;
```

Comment:
In the other file (bt_hal_gatt_types.h) we have the enum values assigned, here they are implied and listed in the comments. Can we please make these 2 consistent. I personally prefer specifying the values instead of having them in the comments as numbers can be incorrect in the comments but this is impossible if values are assigned.

Reply:
I updated this one in the latest commit